fern/openapi-overrides.yml

```diff
@@ -1,10 +1,7 @@
 servers:  
   - url: https://api.givechariot.com
-    description: Production
-    x-fern-server-name: Production
-  - url: https://sandboxapi.givechariot.com
-    description: Sandbox
-    x-fern-server-name: Sandbox
+    description: API
```

Comment:
are we not differentiating between production and sandbox now?

Reply:
What about our API docs?

Reply:
Can I ask why we're making this change? Are we trying to overcome a limitation in the OpenAPI spec or how auth endpoints are different servers than our API endpoints?

Reply:
resolved offline via slack